Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["instant-distance", "instant-distance-py"]
+members = ["distance-metrics", "instant-distance", "instant-distance-py"]
```

Comment:
Curious what your motivation for adding a new crate for this is?

Reply:
Separate mostly because I couldn't get benchmark to access `instant-distance-py` internals and the metrics felt to specific for `instant-distance` crate.

Reply:
I gave this another try. There are 2 things that prevent me from adding benches directly to `instant-distance-py`:
1. crate type is `cdylib` - that one is easy we could have 2 types (`cdylib` and regular `lib`),
2. the name is `instance-distance`, just like the other crate - AFAIK it makes it impossible to use both crates at the same time.

Reply:
It should be possible if we add a tiny python wrapper module. That's to force maturin to build wheel for mixed rust/python project. Without the wrapper maturin uses lib.name as top level module name.